notifier/tasks.py

```diff
@@ -24,7 +24,9 @@
 DEFAULT_LANGUAGE = 'en'
 
 
-@celery.task(rate_limit=settings.FORUM_DIGEST_TASK_RATE_LIMIT, max_retries=settings.FORUM_DIGEST_TASK_MAX_RETRIES)
+@celery.task(rate_limit=settings.FORUM_DIGEST_TASK_RATE_LIMIT,
+             max_retries=settings.FORUM_DIGEST_TASK_MAX_RETRIES,
+             routing_key=settings.ROUTING_KEY)
```

Comment:
Can you add this routing key to any other tasks as well?

Reply:
Just look for the other tasks with '@celery.task'

on them and add the routing key, it is being implicitly set to celery because that is the default, but when we change it, we will want to change both